sdcm/cluster.py

```
@@ -6421,9 +6421,25 @@ def get_node_ip_list(verification_node):
+
+        max_attempts = 10
+        for attempt in range(max_attempts):
+            try:
+                allocator_context = node_allocator.run_nemesis(
+                    nemesis_label="verify decommission", node_list=verification_candidates
+                )
+                verification_node = allocator_context.__enter__()
+                break
+            except AllNodesRunNemesisError:
+                if attempt == max_attempts - 1:
+                    raise AllNodesRunNemesisError("Failed to allocate node for verify_decommission")
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)